internal/crypto/src/tests/raw_signature/validator.rs

```diff
@@ -0,0 +1,121 @@
+// Copyright 2024 Adobe. All rights reserved.
+// This file is licensed to you under the Apache License,
+// Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+// or the MIT license (http://opensource.org/licenses/MIT),
+// at your option.
+
+// Unless required by applicable law or agreed to in writing,
+// this software is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR REPRESENTATIONS OF ANY KIND, either express or
+// implied. See the LICENSE-MIT and LICENSE-APACHE files for the
+// specific language governing permissions and limitations under
+// each license.
+
+#[cfg(target_arch = "wasm32")]
+use wasm_bindgen_test::wasm_bindgen_test;
+
+use crate::{
+    raw_signature::{validator_for_signing_alg, RawSignatureValidationError},
+    SigningAlg,
+};
+
+const SAMPLE_DATA: &[u8] = b"some sample content to sign";
```

Comment:
Could we had tests with different strings? Or vary string used for testing?
I would consider even to go as far as picking strings from those: https://github.com/minimaxir/big-list-of-naughty-strings

Reply:
Mostly because I'm trying to avoid re-creating test assets.